inc/wp-taxonomy-landing/functions/cftl-admin.php

```diff
@@ -447,7 +447,17 @@ function cftl_tax_landing_main($post) {
 		<?php
 			//allow 'custom' if we have a single term
 			$terms = get_the_terms( $post->ID, 'series');
-			if (count($terms) == 1) $series_id = $terms[0]->term_taxonomy_id;
+			/*
+            * issue #1904
+            * when trying to check the count of false we get an error
+            * check its not false and is and array before trying to count it
+            */
+            $terms = get_the_terms( $post->ID, 'series');
+            if ($terms && is_array($terms) && count($terms) == 1) {
```

Comment:
@rlambertsen can you make to sure add spacing here between the arguments and the `( )`?

Also, we'll probably want to make `count($terms) == 1` into `count( $terms ) >= 0` because it's possible that there will be more than one

Reply:
has to be greater than or equal to 1 if. IF zero it will be false and wont get that far